katcp/test/test_resource_client.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Personally I always import print_function, division and absolute_import, to be more sure of getting consistent behaviour between Python 2 and 3. Division is particularly nasty and probably can't be handled by automatic tools since the typing is dynamic.